projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php

```diff
@@ -76,6 +88,15 @@ public function serve() {
 		}
 	}
 
+	/**
+	 * Check if the cache engine has been loaded.
+	 *
+	 * @return bool - True if the cache engine has been loaded, false otherwise.
+	 */
+	public static function is_loading() {
```

Comment:
`is_loading` makes it sound like an ongoing process. Perhaps just `is_loaded` or `has_loaded`?